__tests__/nc-games.test.js

```diff
@@ -30,4 +30,35 @@ describe('GET api/categories', () => {
                 })
             });
     });
-})
\ No newline at end of file
+})
+
+describe.only('GET api/reviews/:review_id', () => {
+    test("returns a review object with properties", () => {
+        return request(app) 
+        .get("/api/reviews/1")
+            .expect(200)
+            .then((res) => {
+                    expect(res.body.review[0]).toEqual(
+                        expect.objectContaining({
+                            review_id: expect.any(Number),
+                            title: expect.any(String),
+                            review_body: expect.any(String),
+                            designer: expect.any(String),
+                            review_img_url: expect.any(String),
+                            votes: expect.any(Number),
+                            category: expect.any(String),
+                            owner: expect.any(String),
+                            created_at: expect.any(String)
+                        })
+                    )
+            });
+    })
+    test("Returns an error if the item does not exist", () => {
+        return request(app)
+            .get("/api/reviews/909090900900")
+            .expect(404)
+            .then((res) => {
+                expect(res.body.message).toBe('not found');
+            });
+    })
```

Comment:
What about when we use an invalid review_id - ie. something like "/api/reviews/notValid"